### PR TITLE
Instant alert handlebars template

### PR DESCRIPTION
**Pull request by debugwand**
* handlebars version for instant alerts button

### CODE REVIEW

File: myft/templates/instant-alert.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		data-trackable="instant"
+		title="Get instant alerts for {{name}}"
+		name="_rel.instant"
+		value="true"
+		type="submit"
+	>{{#if buttonText}}{{buttonText}}{{else}}Instant alerts off{{/if}}</button>
+</form>
+{{else}}
+<!-- Instant alert button hidden due to myFtApiWrite being off -->
+{{/if }}
```
Comment (bjfletcher) [inline myft/templates/instant-alert.html]: trailing space after `/if`

File: myft/templates/instant-alert.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			{{#size}} n-myft-ui__button--{{this}}{{/size}}"
+		data-alternate-label="Stop instant alerts for {{name}}"
+		{{#if alternateText}}
+			data-alternate-text="{{alternateText}}"
+		{{else}}
+		{{#if buttonText}}
+			data-alternate-text="{{buttonText}}"
+		{{else}}
+			data-alternate-text="Instant alerts on"
+		{{/if}}
```
Comment (bjfletcher) [inline myft/templates/instant-alert.html]: indent the inner if to make distinction with outer if clearer

File: myft/templates/instant-alert.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	data-concept-id="{{conceptId}}"
+	action="/__myft/api/core/followed/concept/{{conceptId}}?method=put">
+	<input type="hidden" value="{{name}}" name="name">
+	<input type="hidden" value="{{taxonomy}}" name="taxonomy">
+	<button
+		aria-label="Get instant alerts for {{name}}"
+		aria-pressed="false"
+		class="n-myft-ui__button
+			{{#variant}} n-myft-ui__button--{{this}}{{/variant}}
+			{{#size}} n-myft-ui__button--{{this}}{{/size}}"
```
Comment (bjfletcher) [inline myft/templates/instant-alert.html]: where are the variant & size helpers? (for my understanding)

Comment (debugwand) [inline myft/templates/instant-alert.html]: to match the 'follow' version, you can specify size (follow promo in next-article has this) , and there is an 'inverse' variant which is the dark green colour

**Pull request merged**